clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/provider/view/OpenstackSecurityGroupProvider.groovy

```diff
@@ -50,9 +52,10 @@ class OpenstackSecurityGroupProvider implements SecurityGroupProvider<OpenstackS
     Set<OpenstackSecurityGroup> securityGroups = []
     accountCredentialsProvider.all.each { account ->
       if (account instanceof OpenstackNamedAccountCredentials) {
-        def provider = ((OpenstackNamedAccountCredentials) account).credentials.provider
+        def credentials = ((OpenstackNamedAccountCredentials) account).credentials
+        def provider = credentials.provider
 
-        provider.allRegions.each { region ->
+        provider.getAllRegions().each { region ->
```

Comment:
I am going to leave this as this will be refactored once security groups are loaded into the cache.